sklearn/tests/test_docstring_parameters.py

```diff
@@ -169,6 +169,7 @@ def test_tabs():
                                     % modname)
 
 
+@ignore_warnings(category=FutureWarning)
```

Comment:
I think instead of ignoring the `FutureWarning`s, you could set the random_seed for the estimator the same was as done for `SelectKBest` in this function, with a note to be removed in 0.25. I'd find it a safer approach. WDYT?

Reply:
I'm pretty sure you are right... :)